core/src/main/java/com/linecorp/armeria/client/ClientHttp1ObjectEncoder.java

```diff
@@ -100,6 +100,10 @@ private HttpObject convertHeaders(RequestHeaders headers, boolean endStream) {
         } else {
             nettyHeaders.set(HttpHeaderNames.TRANSFER_ENCODING, HttpHeaderValues.CHUNKED);
         }
+
+        if (headers.contains(HttpHeaderNames.CONNECTION)) {
+            nettyHeaders.set(HttpHeaderNames.CONNECTION, headers.get(HttpHeaderNames.CONNECTION));
+        }
```

Comment:
Can we instead modify `toNettyHttp1ClientHeaders` so it doesn't discard the `connection` header?

Reply:
I checked 'AbstractHttpRequestHandler#writeHeaders' override header by 'ClientRequestContext#additionalRequestHeaders'.
So my code is unnecessary 😇 .